src/components/TabbedNav/TabbedNav.jsx

```diff
@@ -0,0 +1,42 @@
+const React = require('react')
+const hookUpStickyNav = require('../hookUpStickyNav')
+
+class TabbedNav extends React.Component {
+  render() {
+    return (
+      <nav id="tabbed-nav" className="top-nav nav tabs is-toggle is-fullwidth">
+        <ul>
+          <li className={this.props.active === 'tasks' ? 'is-active' : ''}>
+            <a onClick={this.props.showTasks}>
```

Comment:
I disagree - switching it to a button adds a lot of button styling I would have to override, only to make it more semantically correct. Using a `href`-less `<a>` is not ideal, but it works without outside modification.